### PR TITLE
usage example for docs

### PR DESCRIPTION
**Pull request by marioecg**
I added a basic code example for the repository page, might be too barebones but for more elaborated usage the `examples.js` file and folder are great resources.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,64 @@ npm install glsl-sandbox --save
...
+
+#if defined(BACKGROUND)
+    // This will render a red background behind the 3D
+    color.r = 1.0;
+
+#elif defined(POSTPROCESSING)
+    // Postprocessing pass,
+    // displays the scene directly to the screen
+    color = texture2D(u_scene, st);
+
```
Comment (patriciogonzalezvivo) [inline README.md:62]: it's missing an `#else` case here. Which will be use for the geometry material (3D scene).
And yes, 2D scenes don't use `BACKGROUND`

File: README.md [hunk 2]
```diff
@@ -15,8 +15,64 @@ npm install glsl-sandbox --save
...
+void main() {
+    vec4 color = vec4(vec3(0.0), 1.0);
+    vec2 pixel = 1.0 / u_resolution;
+    vec2 st = gl_FragCoord.xy * pixel;
+
+#if defined(BACKGROUND)
+    // This will render a red background behind the 3D
+    color.r = 1.0;
+
+#elif defined(POSTPROCESSING)
```
Comment (patriciogonzalezvivo) [inline README.md:53]: `BACKGROUND` is optional
same with `POSTPROCESSING` on the other example we use it just to display the content of the doubleBuffers

**Comment by marioecg** [comment]
I improved the example based on your last observations, let me know if you think this works better for the docs.

File: README.md [hunk 3]
```diff
@@ -15,8 +15,76 @@ npm install glsl-sandbox --save
...
+    // Postprocessing pass, applies chromatic aberration
+    // and displays the scene directly to the screen
+    color = chromaAB(u_scene, st);
+
+#else
+    // Main shade if it's rendered as a 2D scene
+    // and the shape's surface when rendered a 3D scene 
+    color = texture2D(u_scene, uv);
```
Comment (patriciogonzalezvivo) [inline README.md]: Mmm... in the else is where `u_scene` is render inside the buffer... you can not actually read from it here. 
This is where the geometry is

Comment (marioecg) [reply README.md]: Apologies for that. Just pushed some changes, does that work for you?

**Comment by patriciogonzalezvivo** [comment]
Thank you @marioecg!

I found the example GLSL code a bit confusing and maybe needed a more lengthily explanation. On the other side the JS is very simple with just little differences if you are going to use it for a 2D or a 3D scene with geometry.

Let me know what you think. Feel free to make corrections or suggestions.

**Comment by marioecg** [comment]
You're right @patriciogonzalezvivo! The new documentation you updated is super good in my opinion. It explains in-depth the workflow needed to start working with `GlslSandbox`. I'll read through it again, and if I have any suggestions/corrections, I'll make a PR here 🙂

**Pull request merged**